src/main/java/blue/lapis/pore/impl/PoreServer.java

```diff
@@ -440,7 +464,13 @@ public void savePlayers() {
 
     @Override
     public boolean dispatchCommand(CommandSender sender, String commandLine) throws CommandException {
-        throw new NotImplementedException();
+        Preconditions.checkNotNull(sender, "sender");
+        Preconditions.checkNotNull(commandLine, "commandLine");
+
+        if (commandLine.startsWith("/")) {
```

Comment:
As a long time CraftBukkit user, I can confirm that commands in the format `/cmd` are invalid in the console. You must enter `cmd` - `/cmd` in the console is equivalent to `//cmd` in chat, and `cmd` in the console is equivalent to `/cmd`.

This is especially important when considering plugins that programmatically execute commands, as having the wrong syntax will completely break those plugins.
You can even see it in the help message in the source Minecrell sent; if you're a player, it tells you to use `/help`. If you're the console, it says to use `help`.